test/e2e/e2e_suite_test.go

```diff
@@ -509,8 +509,10 @@ func getLatestSkuForMinor(version string, skus semver.Versions) string {
 			}
 		}
 	} else if v, err := semver.ParseTolerant(version); err == nil {
-		// if the version is in the format "v1.21.2", we make sure we have an existing image for it.
-		Expect(skus).To(ContainElement(v), fmt.Sprintf("Provided Kubernetes version %s does not have a corresponding VM image in the capi offer", version))
+		if len(v.Pre) > 0 {
```

Comment:
should be the opposite no? If v.Pre is empty, then check for a valid image

Reply:
Yep!

🙏

Fixed.